core/templates/services/UpgradedServices.ts

```diff
@@ -700,7 +700,8 @@ export class UpgradedServices {
     upgradedServices['ItemSelectionInputValidationService'] =
       new ItemSelectionInputValidationService(
         upgradedServices['baseInteractionValidationService']);
-    upgradedServices['LocalStorageService'] = new LocalStorageService();
+    upgradedServices['LocalStorageService'] = new LocalStorageService(
+      upgradedServices['WindowRef']);
```

Comment:
Changes to `UpgradedServices.ts` file are not required because these have no effects.

Reply:
Its showing error when I remove `windowRef` from the argument. I have added `windowRef` to `local storage service` newly in this PR, so, we need this change right?